modules/Microsoft.HealthDataServices/deploy.bicep

```diff
@@ -0,0 +1,58 @@
+@description('Required. The name of the Health Data Services Workspace service.')
```

Comment:
Note: The ProviderNamespace folder must be renamed from `Microsoft.HealthDataServices` to `Microsoft.HealthcareApis` as we want to align with the Azure API.